optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessor.java

```diff
@@ -132,6 +134,36 @@ public String createResponseContent() {
         return responseContent.toString();
     }
 
+    public OptOutResults getOptOutResults() {
+        int totalOptedIn = 0;
+        int totalOptedOut = 0;
+
+        try (var dbConnection = DriverManager.getConnection(parameterStore.getDbHost(), parameterStore.getDbUser(), parameterStore.getDbPassword());
+             var statement = dbConnection.createStatement();
+             ResultSet rs = statement.executeQuery(COUNT_STATEMENT)
+        ) {
+            while (rs.next()) {
+                totalOptedIn = rs.getInt("optin");
+                totalOptedOut = rs.getInt("optout");
+            }
+
+            int numberOptedIn = 0;
+            int numberOptedOut = 0;
+
+            for (OptOutInformation optOut : optOutInformationList) {
+                if (Boolean.TRUE.equals(optOut.getOptOutFlag())) {
```

Comment:
Can you please replace this line on ` if (optOut.getOptOutFlag()) { ` ?

Reply:
I had it like: `if (optOut.getOptOutFlag()) {` originally, but 

I needed to change i to satisfy sonarqube. It said that unless we used Boolean.TRUE, it was not null safe. I don't agree, but I had to make this change otherwise it would introduce a code smell according to sonarqube.